src/viewer.js

```diff
@@ -3508,7 +3524,18 @@ function updateOnce( viewer ) {
         abortControlsAutoHide( viewer );
     }
 
-    if ( animated || THIS[ viewer.hash ].forceRedraw || viewer.world.needsDraw() ) {
+    var lastAnimation = false;
+
+    if (currentAnimating) {
+        if (animated) {
+            THIS[ viewer.hash ].animationState = $.ANIMATION_STATES.ANIMATING;
+        } else {
+            THIS[ viewer.hash ].animationState = $.ANIMATION_STATES.ANIMATION_FINISHED;
+            lastAnimation = true;
+        }
+    }
+
+    if ( animated || lastAnimation || THIS[ viewer.hash ].forceRedraw || viewer.world.needsDraw() ) {
```

Comment:
Here the `lastAnimation` is introduced to get `updateOnce` to render again for the last frame where before it was supposed to have skipped that render. The thing is, when setting `animationState` to `$.ANIMATION_STATES.ANIMATION_FINISHED`, it is when `animated` is false, and therefore the next `if` statement protecting the `drawWorld()` call does not enter (both `forceRedraw` and `world.needsDraw()` return false too).